src/components/Hero/Hero.stories.mdx

```diff
@@ -39,7 +38,8 @@ export const enumValues = getStorybookEnumValues(HeroTypes, "HeroTypes");
       table: { defaultValue: { summary: "HeroTypes.Primary" } },
       options: enumValues.options,
     },
-    image: { control: false },
+    imageAlt: { control: false },
```

Comment:
Would it make sense to use one prop, `imageProps`,  here like you did in the Card component?

Reply:
Yep! But this is going to be another ticket/PR because the `Hero` component needs to be refactored: https://jira.nypl.org/browse/DSD-614